code/src/Components/ToDoList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { ToDoItem } from './ToDoItem'
+import { InputToDo } from './InputToDo'
+import { ToDoHeader } from './ToDoHeader'
+import { useSelector } from 'react-redux'
+import { UncontrolledLottie } from './EmptyToDo'
```

Comment:
Try to organize your imports in some way, eg. first the external exports (react, react redux) and then your own components.